exercicios/para-casa/entregas/roxanie-fernandes/ibge.test.js

```diff
@@ -0,0 +1,33 @@
+const request = require("supertest");
```

Comment:
Parabéns novamente aqui. 🥳 Adorei seu teste. Muito bem estruturado e consegui compreender tudo. Como sugestão, poderia, por exemplo, testar como a API responde a situações inesperadas ou inválidas, como acessar rotas inexistentes ou fornecer parâmetros inválidos. Vou deixar um trecho de código de como isso poderia ser feito como exemplo: ` it('Deve retornar status 404 para rota inexistente', async () => {
        const response = await request(api).get('/nonexistent-route');
        expect(response.status).toBe(404);
    });`